prime-router/src/test/resources/fhirengine/smoketest/Expected_FHIR_to_FHIR_MARSOTC.fhir

```diff
@@ -1 +1,1027 @@
\ No newline at end of file
+{
```

Comment:
How does this formatting not break smoketests?

Reply:
All smoke tests involving FHIR bundles are formatting agnostic and are only ensuring structural and data equality. This also ignores expected differences (GUIDs, file creation timestamps, etc). The formatting is simply for reviewer convenience.